conversion/conversion.go

```diff
@@ -213,10 +214,22 @@ func dataFromSQL(driver string, config spanner.BatchWriterConfig, client *sp.Cli
 	return writer, nil
 }
 
+func getDynamoDBClientConfig() aws.Config {
+	cfg := aws.Config{}
+	endpointOverride := os.Getenv("DYNAMODB_ENDPOINT_OVERRIDE")
+	if endpointOverride != "" {
+		cfg = aws.Config{
+			Endpoint: aws.String(endpointOverride),
+		}
```

Comment:
It's more typical to write this as `cfg.Endpoint = aws.String(endpointOverride)` instead of redeclaring cfg.